billchop-fe/src/pages/GroupsPage.tsx

```diff
@@ -54,6 +55,31 @@ export default class GroupsPage extends React.Component<
     this.setState({ selectedGroupId: groupId });
   };
 
+  handleOnAddNewMember = (name: string): void => {
+    const { groups, selectedGroupId } = this.state;
+    const group = groups.find((g) => g.Id === selectedGroupId);
+
+    if (group === undefined) {
+      this.getGroups();
+      return;
+    }
+
+    Axios.post(BASE_URL_API_USERS, {
+      name,
```

Comment:
This currently breaks with new changes added to master (see #21 and #22 ). As a temporary workaround, just to get this code in a working state on master, I suggest doing: ```email: `{name.replace(" ", ".")}@gmail.com` ``` just to have it pass the regex. This should later be fixed in Task 4.